controllers/userController.js

```diff
@@ -150,23 +255,16 @@ const controller = {
       });
     }
     try {
-      const profileOwner = await UserModel.findOne({ username: req.authUser.username });
-      const user = await UserModel.findOne({ username: req.params.username });
-      // authorisation check: whether user is the project owner
-      if (user?._id.toString() !== profileOwner._id.toString()) {
-        return res.status(401).json({
-          error: 'User is not authorised to change this profile',
-        });
-      }
-      const skillsArr = skills
-        ?.split(',')
-        .map((item) => item.trim())
-        .filter((item) => validSkills.includes(item));
-      const interestsArr = interests?.split(',').map((item) => item.trim());
+      // const skillsArr = skills
```

Comment:
can remove this code since commented out

Reply:
updated